pydoctor/templates/apidocs.css

```diff
@@ -195,30 +202,90 @@ tr.private {
     margin-bottom: 6px;
 }
 
+/* This is magic to show the transparent index link */
+a.navbar-brand {
+    /* background-color:#f3d3db; */
+    border:solid;
+    margin-right: 0.3em;
+    border-width: 1.5px;
+    border-color: #c7254e;
+    background-color:#f9f2f4;
+    border-radius: 4px;
+}
+
+/* This is for the main top navagation bar */
+.page-header > h1 > code {
+    background-color:#f9f2f4;
+    /* background-color:transparent; */
+}
+
+/* .page-header > h1 > code > a {
+    background-color:#FCF9FA;
+} */
+
+/* 
+<a><code></code><a> for links
+This is inline docstring content marked up as code, for example C{foo} in epytext
+*/
+a > code {
+    color:#c7254e; 
+    background-color:#f9f2f4;
+    /* background-color:transparent; */
+}
+/* 
+<a class="code"> links that are present in summary tables 
+*/
+a.code {
```

Comment:
If we use the same styling in multiple places, we can use the `a, b { ... }` selector syntax to avoid duplicating the rules.